website/docs/r/policy_profile_enrollment_apps.html.markdown

```diff
@@ -8,7 +8,7 @@ description: |-
 
 # okta_policy_profile_enrollment_apps
 
-~> **WARNING:** This feature is only available as a part of the Identity Engine. [Contact support](mailto:dev-inquiries@okta.com) for further information.
+~> **WARNING:** This feature does not work with orgs having the feature flag `SHARED_APP_SIGN_ON_POLICIES` which may include orgs not fully upgraded to Identity Engine. [Contact support](mailto:dev-inquiries@okta.com) for further information.
```

Comment:
Are you sure it's for orgs that have the FF `SHARED_APP_SIGN_ON_POLICIES`? or is it org that do NOT have `SHARED_APP_SIGN_ON_POLICIES`?

Reply:
I'll undo that text. I'm not 100% on the claim now that I double checked it.